modules/nextflow/src/main/groovy/nextflow/executor/BashWrapperBuilder.groovy

```diff
@@ -484,6 +484,9 @@ class BashWrapperBuilder {
         if( this.containerMemory )
             builder.setMemory(containerMemory)
 
+        if( this.containerCpus )
+            builder.setCpus(containerCpus)
+
```

Comment:
@pditommaso , is this not what you meant? I updated the `BashWrapperBuilder` when you mentioned earlier.

Reply:
You are right! 🎉